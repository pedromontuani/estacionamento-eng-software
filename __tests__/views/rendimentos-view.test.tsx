import React from 'react';
import {rendererCreate} from '../utils';

import RendimentosView from '../../src/views/rendimentos-view';

describe('Testing users view', () => {
  test('should match snapshot', async () => {
    const tree = rendererCreate(<RendimentosView />);
    expect(tree).toMatchSnapshot();
  });
});
