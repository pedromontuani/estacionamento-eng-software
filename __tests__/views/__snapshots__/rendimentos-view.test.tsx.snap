// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing users view should match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "flex": 1,
      "padding": 24,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderRadius": 16,
        "elevation": 5,
        "justifyContent": "center",
        "marginVertical": 24,
        "paddingVertical": 12,
      }
    }
  >
    <Text>
      Rendimento mensal
    </Text>
    <View
      style={
        Object {
          "borderRadius": 16,
          "marginVertical": 8,
        }
      }
    >
      <RNSVGSvgView
        bbHeight={220}
        bbWidth={670}
        focusable={false}
        height={220}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "flex": 0,
              "height": 220,
              "width": 670,
            },
          ]
        }
        width={670}
      >
        <RNSVGGroup>
          <RNSVGDefs>
            <RNSVGLinearGradient
              gradient={
                Array [
                  0,
                  -1,
                  1,
                  -1,
                ]
              }
              gradientTransform={null}
              gradientUnits={1}
              name="backgroundGradient"
              x1={0}
              x2={670}
              y1={220}
              y2={0}
            />
            <RNSVGLinearGradient
              gradient={
                Array [
                  0,
                  436207616,
                  1,
                  436207616,
                ]
              }
              gradientTransform={null}
              gradientUnits={1}
              name="fillShadowGradientFrom"
              x1={0}
              x2={0}
              y1={0}
              y2={220}
            />
          </RNSVGDefs>
          <RNSVGDefs />
          <RNSVGRect
            fill={
              Array [
                1,
                "backgroundGradient",
              ]
            }
            height={220}
            propList={
              Array [
                "fill",
              ]
            }
            rx={16}
            ry={16}
            width="100%"
            x={0}
            y={0}
          />
          <RNSVGGroup>
            <RNSVGLine
              propList={
                Array [
                  "stroke",
                  "strokeWidth",
                  "strokeDasharray",
                ]
              }
              stroke={855638016}
              strokeDasharray={
                Array [
                  "5",
                  "10",
                ]
              }
              strokeWidth={1}
              x1={0}
              x2={670}
              y1={16}
              y2={16}
            />
            <RNSVGLine
              propList={
                Array [
                  "stroke",
                  "strokeWidth",
                  "strokeDasharray",
                ]
              }
              stroke={855638016}
              strokeDasharray={
                Array [
                  "5",
                  "10",
                ]
              }
              strokeWidth={1}
              x1={0}
              x2={670}
              y1={57.25}
              y2={57.25}
            />
            <RNSVGLine
              propList={
                Array [
                  "stroke",
                  "strokeWidth",
                  "strokeDasharray",
                ]
              }
              stroke={855638016}
              strokeDasharray={
                Array [
                  "5",
                  "10",
                ]
              }
              strokeWidth={1}
              x1={0}
              x2={670}
              y1={98.5}
              y2={98.5}
            />
            <RNSVGLine
              propList={
                Array [
                  "stroke",
                  "strokeWidth",
                  "strokeDasharray",
                ]
              }
              stroke={855638016}
              strokeDasharray={
                Array [
                  "5",
                  "10",
                ]
              }
              strokeWidth={1}
              x1={0}
              x2={670}
              y1={139.75}
              y2={139.75}
            />
            <RNSVGLine
              propList={
                Array [
                  "stroke",
                  "strokeWidth",
                  "strokeDasharray",
                ]
              }
              stroke={855638016}
              strokeDasharray={
                Array [
                  "5",
                  "10",
                ]
              }
              strokeWidth={1}
              x1={0}
              x2={670}
              y1={181}
              y2={181}
            />
          </RNSVGGroup>
          <RNSVGGroup>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "end",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  52,
                ]
              }
              y={
                Array [
                  181,
                ]
              }
            >
              <RNSVGTSpan
                content="$0.00"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "end",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  52,
                ]
              }
              y={
                Array [
                  139.75,
                ]
              }
            >
              <RNSVGTSpan
                content="$0.25"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "end",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  52,
                ]
              }
              y={
                Array [
                  98.5,
                ]
              }
            >
              <RNSVGTSpan
                content="$0.50"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "end",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  52,
                ]
              }
              y={
                Array [
                  57.25,
                ]
              }
            >
              <RNSVGTSpan
                content="$0.75"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "end",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  52,
                ]
              }
              y={
                Array [
                  16,
                ]
              }
            >
              <RNSVGTSpan
                content="$1.00"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
          </RNSVGGroup>
          <RNSVGGroup>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "middle",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  96,
                ]
              }
              y={
                Array [
                  205,
                ]
              }
            >
              <RNSVGTSpan
                content="Janeiro"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "middle",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  298,
                ]
              }
              y={
                Array [
                  205,
                ]
              }
            >
              <RNSVGTSpan
                content="Fevereiro"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "middle",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  500,
                ]
              }
              y={
                Array [
                  205,
                ]
              }
            >
              <RNSVGTSpan
                content="Março"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
          </RNSVGGroup>
          <RNSVGGroup>
            <RNSVGRect
              fill={
                Array [
                  1,
                  "fillShadowGradientFrom",
                ]
              }
              height={0}
              propList={
                Array [
                  "fill",
                ]
              }
              rx={0}
              width={32}
              x={80}
              y={181}
            />
            <RNSVGRect
              fill={
                Array [
                  1,
                  "fillShadowGradientFrom",
                ]
              }
              height={0}
              propList={
                Array [
                  "fill",
                ]
              }
              rx={0}
              width={32}
              x={282}
              y={181}
            />
            <RNSVGRect
              fill={
                Array [
                  1,
                  "fillShadowGradientFrom",
                ]
              }
              height={0}
              propList={
                Array [
                  "fill",
                ]
              }
              rx={0}
              width={32}
              x={484}
              y={181}
            />
          </RNSVGGroup>
          <RNSVGGroup />
          <RNSVGGroup>
            <RNSVGRect
              fill={2566914048}
              height={2}
              propList={
                Array [
                  "fill",
                ]
              }
              width={32}
              x={80}
              y={181}
            />
            <RNSVGRect
              fill={2566914048}
              height={2}
              propList={
                Array [
                  "fill",
                ]
              }
              width={32}
              x={282}
              y={181}
            />
            <RNSVGRect
              fill={2566914048}
              height={2}
              propList={
                Array [
                  "fill",
                ]
              }
              width={32}
              x={484}
              y={181}
            />
          </RNSVGGroup>
        </RNSVGGroup>
      </RNSVGSvgView>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderRadius": 16,
        "elevation": 5,
        "justifyContent": "center",
        "marginVertical": 24,
        "paddingVertical": 12,
      }
    }
  >
    <Text>
      Rendimento semanal
    </Text>
    <View
      style={
        Object {
          "borderRadius": 16,
          "marginVertical": 8,
        }
      }
    >
      <RNSVGSvgView
        bbHeight={220}
        bbWidth={670}
        focusable={false}
        height={220}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "borderWidth": 0,
            },
            Object {
              "flex": 0,
              "height": 220,
              "width": 670,
            },
          ]
        }
        width={670}
      >
        <RNSVGGroup>
          <RNSVGDefs>
            <RNSVGLinearGradient
              gradient={
                Array [
                  0,
                  -1,
                  1,
                  -1,
                ]
              }
              gradientTransform={null}
              gradientUnits={1}
              name="backgroundGradient"
              x1={0}
              x2={670}
              y1={220}
              y2={0}
            />
            <RNSVGLinearGradient
              gradient={
                Array [
                  0,
                  436207616,
                  1,
                  436207616,
                ]
              }
              gradientTransform={null}
              gradientUnits={1}
              name="fillShadowGradientFrom"
              x1={0}
              x2={0}
              y1={0}
              y2={220}
            />
          </RNSVGDefs>
          <RNSVGDefs />
          <RNSVGRect
            fill={
              Array [
                1,
                "backgroundGradient",
              ]
            }
            height={220}
            propList={
              Array [
                "fill",
              ]
            }
            rx={16}
            ry={16}
            width="100%"
            x={0}
            y={0}
          />
          <RNSVGGroup>
            <RNSVGLine
              propList={
                Array [
                  "stroke",
                  "strokeWidth",
                  "strokeDasharray",
                ]
              }
              stroke={855638016}
              strokeDasharray={
                Array [
                  "5",
                  "10",
                ]
              }
              strokeWidth={1}
              x1={0}
              x2={670}
              y1={16}
              y2={16}
            />
            <RNSVGLine
              propList={
                Array [
                  "stroke",
                  "strokeWidth",
                  "strokeDasharray",
                ]
              }
              stroke={855638016}
              strokeDasharray={
                Array [
                  "5",
                  "10",
                ]
              }
              strokeWidth={1}
              x1={0}
              x2={670}
              y1={57.25}
              y2={57.25}
            />
            <RNSVGLine
              propList={
                Array [
                  "stroke",
                  "strokeWidth",
                  "strokeDasharray",
                ]
              }
              stroke={855638016}
              strokeDasharray={
                Array [
                  "5",
                  "10",
                ]
              }
              strokeWidth={1}
              x1={0}
              x2={670}
              y1={98.5}
              y2={98.5}
            />
            <RNSVGLine
              propList={
                Array [
                  "stroke",
                  "strokeWidth",
                  "strokeDasharray",
                ]
              }
              stroke={855638016}
              strokeDasharray={
                Array [
                  "5",
                  "10",
                ]
              }
              strokeWidth={1}
              x1={0}
              x2={670}
              y1={139.75}
              y2={139.75}
            />
            <RNSVGLine
              propList={
                Array [
                  "stroke",
                  "strokeWidth",
                  "strokeDasharray",
                ]
              }
              stroke={855638016}
              strokeDasharray={
                Array [
                  "5",
                  "10",
                ]
              }
              strokeWidth={1}
              x1={0}
              x2={670}
              y1={181}
              y2={181}
            />
          </RNSVGGroup>
          <RNSVGGroup>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "end",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  52,
                ]
              }
              y={
                Array [
                  181,
                ]
              }
            >
              <RNSVGTSpan
                content="$0.00"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "end",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  52,
                ]
              }
              y={
                Array [
                  139.75,
                ]
              }
            >
              <RNSVGTSpan
                content="$0.25"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "end",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  52,
                ]
              }
              y={
                Array [
                  98.5,
                ]
              }
            >
              <RNSVGTSpan
                content="$0.50"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "end",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  52,
                ]
              }
              y={
                Array [
                  57.25,
                ]
              }
            >
              <RNSVGTSpan
                content="$0.75"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "end",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  52,
                ]
              }
              y={
                Array [
                  16,
                ]
              }
            >
              <RNSVGTSpan
                content="$1.00"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
          </RNSVGGroup>
          <RNSVGGroup>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "middle",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  96,
                ]
              }
              y={
                Array [
                  205,
                ]
              }
            >
              <RNSVGTSpan
                content="01 - 08"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "middle",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  247.5,
                ]
              }
              y={
                Array [
                  205,
                ]
              }
            >
              <RNSVGTSpan
                content="09 - 16"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "middle",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  399,
                ]
              }
              y={
                Array [
                  205,
                ]
              }
            >
              <RNSVGTSpan
                content="17 - 22"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
            <RNSVGText
              content={null}
              dx={Array []}
              dy={Array []}
              fill={3422552064}
              font={
                Object {
                  "fontSize": 12,
                  "textAnchor": "middle",
                }
              }
              matrix={
                Array [
                  1,
                  0,
                  -0,
                  1,
                  0,
                  0,
                ]
              }
              propList={
                Array [
                  "fill",
                ]
              }
              rotate={Array []}
              x={
                Array [
                  550.5,
                ]
              }
              y={
                Array [
                  205,
                ]
              }
            >
              <RNSVGTSpan
                content="23 - 31"
                dx={Array []}
                dy={Array []}
                font={Object {}}
                rotate={Array []}
                x={Array []}
                y={Array []}
              />
            </RNSVGText>
          </RNSVGGroup>
          <RNSVGGroup>
            <RNSVGRect
              fill={
                Array [
                  1,
                  "fillShadowGradientFrom",
                ]
              }
              height={0}
              propList={
                Array [
                  "fill",
                ]
              }
              rx={0}
              width={32}
              x={80}
              y={181}
            />
            <RNSVGRect
              fill={
                Array [
                  1,
                  "fillShadowGradientFrom",
                ]
              }
              height={0}
              propList={
                Array [
                  "fill",
                ]
              }
              rx={0}
              width={32}
              x={231.5}
              y={181}
            />
            <RNSVGRect
              fill={
                Array [
                  1,
                  "fillShadowGradientFrom",
                ]
              }
              height={0}
              propList={
                Array [
                  "fill",
                ]
              }
              rx={0}
              width={32}
              x={383}
              y={181}
            />
            <RNSVGRect
              fill={
                Array [
                  1,
                  "fillShadowGradientFrom",
                ]
              }
              height={0}
              propList={
                Array [
                  "fill",
                ]
              }
              rx={0}
              width={32}
              x={534.5}
              y={181}
            />
          </RNSVGGroup>
          <RNSVGGroup />
          <RNSVGGroup>
            <RNSVGRect
              fill={2566914048}
              height={2}
              propList={
                Array [
                  "fill",
                ]
              }
              width={32}
              x={80}
              y={181}
            />
            <RNSVGRect
              fill={2566914048}
              height={2}
              propList={
                Array [
                  "fill",
                ]
              }
              width={32}
              x={231.5}
              y={181}
            />
            <RNSVGRect
              fill={2566914048}
              height={2}
              propList={
                Array [
                  "fill",
                ]
              }
              width={32}
              x={383}
              y={181}
            />
            <RNSVGRect
              fill={2566914048}
              height={2}
              propList={
                Array [
                  "fill",
                ]
              }
              width={32}
              x={534.5}
              y={181}
            />
          </RNSVGGroup>
        </RNSVGGroup>
      </RNSVGSvgView>
    </View>
  </View>
</View>
`;
